### ISSUE TITLE
Supporting multiple assets with the same name

### ISSUE THREAD

**Issue by egbutter** [NREL-SIIP/PowerSystems.jl#687]
I attempted to parse a PSSE raw file, and the parser raised an error that a bus with that name already exists. The RAW file does have the same name for multiple buses, but this is expected. System files sometimes do repeat the same name for multiple buses tied to the same asset, and PSSE allows this. Rather than raising a hard error, it would be more useful to include a flag to append the bus number / uuid. 

```
sys =  System(PowerModelsData(joinpath(home_dir, "XXXX.raw")));
ArgumentError: XXXX     is already stored for type Bus
```

**Comment by claytonpbarrows** [comment]
You should be able to do:
```
sys = System(joinpath(home_dir, "XXXX.raw"), bus_name_formatter = x -> string(x["name"]) * "-" * string(x["index"]))
```

Sorry, this feature is not well documented.

**Comment by jd-lara** [comment]
@claytonpbarrows and @daniel-thom can we add a better error message about this?

**Comment by egbutter** [comment]
@claytonpbarrows thanks for the tip. This solved the immediate problem. I'm still not able to parse this file due to other issues I will need to investigate myself, I think.

**Comment by jd-lara** [comment]
closed via #690
